marlowe-dashboard-client/src/Pickup/State.purs

```diff
@@ -99,3 +102,10 @@ handleAction PickupWallet = do
   walletLibrary <- use _walletLibrary
   liftEffect $ setItem walletLibraryLocalStorageKey $ encodeJSON walletLibrary
   callMainFrameAction $ MainFrame.EnterPlayState walletLibrary walletDetails
+
+handleAction ClearLocalStorage = do
+  liftEffect $ removeItem walletLibraryLocalStorageKey
+  liftEffect $ removeItem walletDetailsLocalStorageKey
+  window_ <- liftEffect window
+  location_ <- liftEffect $ location window_
```

Comment:
very minor, but 

```purescript
location_ <- liftEffect <<< location <<< window
```

Reply:
Compiler says no. :confused: I feel there must be a way to simplify this, but for some reason I can't see what it is at the moment.

Reply:
sorry, my bad :P 

```purescript
handleAction ClearLocalStorage = do
  liftEffect do
    removeItem walletLibraryLocalStorageKey
    removeItem walletDetailsLocalStorageKey
  location_ <- liftEffect $ location =<< window
  liftEffect $ reload location_
```